p2panda-core/src/operation.rs

```diff
@@ -1,19 +1,115 @@
 // SPDX-License-Identifier: AGPL-3.0-or-later
 
-use serde::de::DeserializeOwned;
-use serde::Serialize;
+//! Core p2panda data type offering distributed, secure and efficient data transfer between peers.
```

Comment:
These are the first docs I wrote in this PR and I based it of some already existing texts. I wrote the `lib.rs` texts last and now feel that they are too different in content and tone to what I have here.... I'd like to make them more consistent, probably editing this one to match `lib.rs` more, but going into more detail.